_config.yml

```diff
@@ -115,61 +115,38 @@ highlighter: rouge
 #     output: false
 #
 collections:
-  how-to-use-guide:
-    title: "How to use this guide"
-    description: "About this Content Guide and how you can use it."
-    order: 0
-    output: true
   content-structure:
     title: "Content structure"
     description: "The basic building blocks for crafting good content."
     order: 1
-    output: true
```

Comment:
I removed these outputs, as the resulting pages weren't used as far as I could see and the link checker was failing them (i.e. the content in some of the collection items was expecting to be on the collection index page)